users/users.go

```diff
@@ -0,0 +1,69 @@
+package users
+
+import (
+	"encoding/json"
+	"errors"
+	"io/ioutil"
+	"os"
+)
+
+var ErrNotFound = errors.New("User not found")
```

Comment:
Could be a good idea to have constant Errors

Reply:
The language does not allow to have constants of pointers, and `errors.New()` returns a pointer.

Reply:
Thank you for the explanation.